getting-set-up.qmd

```diff
@@ -79,3 +53,32 @@ cmdstanr::cmdstan_version()
 ```
 
 For more details, and for links to resources in case something goes wrong, see the [Getting Started with CmdStanr](https://mc-stan.org/cmdstanr/articles/cmdstanr.html) vignette of the package.
+
+# Accessing the course
+
+To be able to use the code in each session, you will need a local copy of the course material.
+
+-   Directly download the course material:
+
+    ::: callout-tip
+    [[**Download**]{.underline}](https://github.com/nfidd/nfidd/archive/refs/heads/main.zip)
+    :::
+
+-   Alternatively, if you are familiar with git you can clone the [repo](https://github.com/nfidd/nfidd).
+
+### Interacting with the course
+
+In this course, all content is written using [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+This means that we can combine text with code and see the output directly.
+The notebooks are then directly reproduced on the course website (for example, this page).
+
+To interact with each session in the course, we recommend opening the RStudio Project file (`nfidd.RProj`) in the `nfidd` folder you have just downloaded.
```

Comment:
Why do we ask them to download at all for the version of this where they are copying and pasting? It's an or isn't it?